release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -121,6 +121,35 @@ io:println(nameText); // "DanBrown"
 
 ##### Language Server
 
+##### Ballerina Shell
+
+- Ballerina Shell now supports redefining module-level definitions and variable declarations. 
```

Comment:
Shall we add the example here?
````suggestion
- Ballerina Shell now supports redefining module-level definitions and variable declarations. 
```ballerina
=$ int i = 3;
=$ string i = "Hello";  // Same variable can be redefined
=$ i
"Hello"
```
````